gprotorch/kernels/fingerprint_kernels/tanimoto_kernel.py

```diff
@@ -3,43 +3,56 @@
 RDKit fragment features.
 """
 
+import torch
 import gpytorch
 
-from gprotorch.kernels.fingerprint_kernels.base_fingerprint_kernel import BitKernel
+from gprotorch.kernels.fingerprint_kernels.base_fingerprint_kernel import (
```

Comment:
What do the brackets do in the import statement?

Reply:
This was put there by black. It allows imports to be put over multiple lines.